src/utils/indexForSearch.ts

```diff
@@ -10,18 +10,37 @@ import { search } from "../lib/search"
  * This indexes an ElasticSearch cluster with collections data.
  */
 
+interface DebugRecord {
```

Comment:
This is just a type so that the compiler stops yelling at me - I want to log out the set of all records that had any kind of issue and so were unable to be updated.